Cargo.toml

```diff
@@ -25,27 +25,27 @@ tag-prefix = ""
 [dependencies]
 base64 = "0.11"
 error-chain = { version = "0.12", default-features = false }
-futures = "0.1"
-http = "0.1"
+futures = "0.3"
+http = "0.2"
 libflate = "0.1"
 log = "0.4"
 mime = "0.3"
 mockito = { version = "0.22", optional = true }
 regex = "^1.1.0"
-serde = "1"
-serde_derive = "1"
+serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 strum = "0.16"
 strum_macros = "0.16"
 tar = "0.4"
-tokio = "0.1"
+tokio = "0.2"
 dirs = "2.0"
-reqwest = { version = "^0.9.6", default-features = false }
+reqwest = { version = "0.10", default-features = false, features = ["json"] }
 sha2 = "^0.8.0"
 
 [dev-dependencies]
 env_logger = "0.7"
 spectral = "0.6"
+tokio = { version = "0.2", features = ["macros"] }
```

Comment:
To the best of my knowledge, cargo will do a union of the features-set between this and the other `tokio` dependency.

You can either promote the `macros` feature to the other dependency, or avoid using macros in examples (my preference is for the latter). At that point, you can simply drop this line.

Reply:
I think and tested that dev-dependencies won't be exposed to users of this crate. If I try to use following code in a rust application, the compiler tells me that it cannot find the code that is only available behind the feature `macros`.

```rust
#[tokio::main]
async fn main() -> Result<(), Error> {
    // …
}
```

```
|
| #[tokio::main]
|          ^^^^ could not find `main` in `tokio`
```